client/src/App.js

```diff
@@ -1,52 +1,57 @@
-import React from "react";
+import React, { useState } from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
 import { theme } from "./themes/theme";
-import Login from "./Components/LoginForm";
-import SignUp from "./Components/SignupForm";
+
+import Login from "./pages/LoginForm";
+import SignUp from "./pages/SignupForm";
 // import LandingPage from "./pages/Landing";
 import Profile from "./pages/Profile";
 // import ShareMusic from "./pages/ShareMusic";
 import Discover from "./pages/Discover";
 import Messages from "./pages/Messages";
 import Nav from "./Components/Nav";
-import SingularPost from "./Components/SingularPost";
 import "./App.css";
+import ProfileUpdate from "./pages/ProfileUpdate";
 
 
 function App() {
-  // when true, user logged in and will not be redirected to dashboard
-  let userLoggedIn = false;
-
-  let loggedIn = (
+  const [auth, setAuth] = useState({
```

Comment:
I think this whole logic could be a custom hook that is declared elsewhere. 
Read this: https://reactjs.org/docs/hooks-custom.html